packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/util.ts

```diff
@@ -8,6 +8,8 @@ import {
 } from '@cardano-sdk/core';
 import BigNumber from 'bignumber.js';
 
+export const REWARDS_HISTORY_LIMIT_DEFAULT = 3;
```

Comment:
Depending on which PR is merged first, let's make this a parameter and use the config to set the default limit to `0` should APY metric be disabled. See @iccicci's comment here: https://github.com/input-output-hk/cardano-js-sdk/pull/609#issuecomment-1435940833

Reply:
#609 is now on hold, so please ignore my previous comment, and we can handle this change later if needed